tests/testapps/CGCatalog/CGCatalog.vsimporter/CGCatalog-WinStore10/CGCatalog.vcxproj.filters

```diff
@@ -202,6 +202,8 @@
     <SBResourceCopy Include="..\..\CGCatalog\Resources\CloseSubPath.png" />
     <SBResourceCopy Include="..\..\CGCatalog\Resources\GetBoundingBox.png" />
     <SBResourceCopy Include="..\..\CGCatalog\Resources\CloseSubPath.png" />
+    <SBResourceCopy Include="..\..\CGCatalog\Resources\GetBoundingBox.png" />
+    <SBResourceCopy Include="..\..\CGCatalog\Resources\CloseSubPath.png" />
```

Comment:
can we just not copy over everything in the directory?
That way we don't have to keep on expanding this.

  <ItemGroup>
    <SBResourceCopy Include="..\..\CGCatalog\Resources\*" />
</ItemGroup>